checklist.md

```diff
@@ -4,9 +4,16 @@
 
 Advance:
 
+1. rkdarst: test microphones
 1. Write session outline
+   - First line of each section is copy-pasteable to youtube TOC,
+     including links.
```

Comment:
can we add an example here on how this should ideally look? one entry.

Reply:
Updated, is this good?